services/director-v2/src/simcore_service_director_v2/modules/director_v0.py

```diff
@@ -118,3 +135,35 @@ async def get_running_service_details(
         if resp.status_code == status.HTTP_200_OK:
             return RunningServiceDetails.parse_obj(unenvelope_or_raise_error(resp))
         raise HTTPException(status_code=resp.status_code, detail=resp.content)
+
+    @log_decorator(logger=logger)
+    async def get_service_labels(self, service: ServiceKeyVersion) -> SimcoreService:
+        resp = await self.request(
+            "GET",
+            f"services/{urllib.parse.quote_plus(service.key)}/{service.version}:labels",
```

Comment:
```suggestion
            f"services/{urllib.parse.quote_plus(service.key)}/{service.version}/labels",
```

not that labels are a resource. therefore not a custom function

Reply:
will replace with `/`